vagrant/provisioning/roles/arkcase-app/templates/server.yml

```diff
@@ -145,11 +145,11 @@ payment:
 email:
   #set some existing email here for testing
   CASE_FILE.user: "{{ email_receiver_account_escaped | default('') }}"
-  CASE_FILE.password: "ENC({{ encrypted_email_receiver_password | default('') }})"
+  CASE_FILE.password: "ENC({{ encrypted_email_receiver_password }})"
   COMPLAINT.user: "{{ email_receiver_account_escaped | default('') }}"
-  COMPLAINT.password: "ENC({{ encrypted_email_receiver_password | default('') }})"
+  COMPLAINT.password: "ENC({{ encrypted_email_receiver_password }})"
   CONSULTATION.user: "{{ email_receiver_account_escaped | default('') }}"
-  CONSULTATION.password: "ENC({{ encrypted_email_receiver_password | default('') }})"
+  CONSULTATION.password: "ENC({{ encrypted_email_receiver_password }})"
```

Comment:
Now I see other issues. All these must be separate accounts. So, you would have:
email_receiver_case_file_account_escaped
email_receiver_complaint_account_escaped
email_receiver_consultation_account_escaped

Same for passwords.